cdap-app-templates/cdap-program-report/src/test/java/io/cdap/cdap/report/ReportGenerationAppTest.java

```diff
@@ -112,7 +112,7 @@ public class ReportGenerationAppTest extends TestBase {
   public static final TemporaryFolder TEMP_FOLDER = new TemporaryFolder();
   @ClassRule
   public static final TestConfiguration CONF =
-    new TestConfiguration(io.cdap.cdap.common.conf.Constants.Explore.EXPLORE_ENABLED, "false");
+    new TestConfiguration();
```

Comment:
You don't need this rule if you don't have custom settings

Reply:
Removed